src/ResourceManager.ForRefactoringOnly.sln

```diff
@@ -130,6 +133,7 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Commands.DataLakeStore.Test", "ResourceManager\DataLakeStore\Commands.DataLakeStore.Test\Commands.DataLakeStore.Test.csproj", "{B0D03ECF-9F25-499A-BE25-D668E0D208AA}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Commands.DataLakeAnalytics.Test", "ResourceManager\DataLakeAnalytics\Commands.DataLakeAnalytics.Test\Commands.DataLakeAnalytics.Test.csproj", "{E6122DB1-1466-47EE-8BA0-73F9CA90F826}"
+>>>>>>> refs/remotes/Azure/dev
```

Comment:
Why do you have this in the .sln file? It looks like the result of github conflict resolution.